### ISSUE TITLE
Fix test result on Example 1 in check_hierarchy

### ISSUE THREAD

**Issue by javihern98** [VTL-Community/VTL-Community#55]
Checking the result of check_hierarchy example, we found some issues:

Hierarchical Ruleset HR_1:
![image](https://github.com/user-attachments/assets/d8955c0c-214f-4295-889a-5a5309d7ff69)

Input Dataset DS_1:
![image](https://github.com/user-attachments/assets/972b6fb6-a566-4748-939f-1795d4e29ac7)

Result:
![image](https://github.com/user-attachments/assets/1c2915d9-e51c-47f9-ac2b-f32423fba74e)

The expected result defined by the Example is wrong when Id_2 has these values: E, F and H (with mode partial_null, as specified in the example):

## E value (solved already on version 2.1)

It is not defined on the Hierarchical Ruleset. We believe it should be defined as: 

```sql
E = T + U + V errorlevel 0
```
 Taking into consideration the Hierarchical ruleset defined in hierarchy operator (line 5302) and the result of the Example (error level is defined as 0):
![image](https://github.com/user-attachments/assets/1958772c-42d4-437d-b65f-3ee81543f546)

## E, F and H value

It should not be present due to the behavior of the partial_null mode (no datapoint is present on left or right side of the comparison,
therefore the rule should not be evaluated and it should not appear in the result):
![image](https://github.com/user-attachments/assets/b7b9cb14-64b9-4500-afa2-8ec21281fe85)

**Comment by javihern98** [comment]
@linardian Please check it

**Comment by linardian** [comment]
Please refer to the updated version!!! The typo had already been corrected (definition of rule for E)...

**Comment by javihern98** [comment]
@linardian Yes, still the E, F and H values should not be present, they do also appear on version 2.1

**Comment by javihern98** [comment]
@linardian Okay, still values F and H need to be checked

**Comment by linardian** [comment]
Well:

Since:
E = T + U + V				errorlevel 0  
--> the E line should be present.
F = Y + W + Z 			errorlevel 7
--> F should be deleted
H = D + E 
D = R + S 
E = T + U + V
--> H should be present

So I would cancel the F line from the resulting table: is that right ?

**Comment by javihern98** [comment]
Depends if we apply the partial_null behaviour we agreed on #54 also for check_hierarchy. If so, we should change it also in check_hierarchy and then it seems consistent.

**Comment by linardian** [comment]
Yes. I corrected also the corrisponding rows in the table consistently with the table in the hierarchy operator (If at least one of the involved Data Points **exixts**)
